run/system-package/e2e_test_setup.yaml

```diff
@@ -0,0 +1,40 @@
+# Copyright 2020 Google, LLC.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+steps:
+- # Build the renderer image
+  name: gcr.io/cloud-builders/docker:latest
+  args: ['build', '--tag=gcr.io/$PROJECT_ID/sys-package-${_SUFFIX}', '.']
```

Comment:
Instead of using _SUFFIX here, can we use $COMMIT_SHA as the image tag?

Reply:
Hrm - I have to get the URL in the test and to do that I need the name of the service. Not sure how I would get the COMMIT SHA into pytest to get the URL?

Reply:
Maybe you can just use the `${_SUFFIX}` as the tag, instead of a part of the image name?

Reply:
If it's just part of the image name and not the service, wouldn't I create problems by deploying the image to the same service name if other tests are also running?  Wouldn't I overwrite it?

Reply:
You need to keep the service name as `sys-package-${_SUFFIX}`.

Reply:
Looks like COMMIT_SHA only works on builds that are triggered by a commit. Since I'm explicitly calling the build in the test, it doesn't have the substitution available and is null.   

Results in error: 

`Step #0: invalid argument "gcr.io/python-docs-samples-tests/sys-package:" for "-t, --tag" flag: invalid reference format`